.gitignore

```diff
@@ -1,6 +1,7 @@
 *~
 *swp
 *.charm
+.coverage
```

Comment:
I think we want to have everyone using the same coverage settings so we should probably commit this file

Reply:
This is an autogenerated coverage report. I do not see why it should be committed.

Reply:
Ah I see. I thought this was .coveragerc

It's a little bit weird they would start the report file with "."